docs/ui/schema_viewer.rst

```diff
@@ -0,0 +1,76 @@
+.. _ref_ui_schema_viewer:
+
+=============
+Schema Viewer
+=============
+
+The UI schema viewer allows you to see your schema either as text, graph,
+or both.
+
+Text view
+---------
+
+The UI's text view is similar to what is produced in the CLI through the
+``describe schema as sdl`` command, but includes inherited properties and
+links.
+
+The schema viewer shows the user's defined schema by default under the
+name ``User``, and also includes ``Stdlib`` and ``System`` as a convenience.
+Along with the schema viewer's built-in search function, this makes
+the schema viewer often even more convenient than searching through EdgeDB's
+online documentation.
+
+For example, a search for 'max val' immediately shows the following output:
+
+.. code-block::
+
+    abstract constraint std::max_value(max: anytype) {
+    using ((__subject__ <= max));
+    }  errmessage := 
+        'Maximum allowed value for {__subject__} is {max}.';
+    annotation std::description := 
+        'Specifies the maximum value for the subject.';
+    ;
+
+    abstract constraint std::max_ex_value(max: anytype) {
+    using ((__subject__ < max));
+    }  errmessage := 
+        '{__subject__} must be less than {max}.';
+    annotation std::description := 
+        'Specifies the maximum value (as an open interval) for the subject.';
+    ;
+
+    abstract constraint std::max_len_value(max: std::int64) 
+        on (std::len(<std::str>__subject__)) {
+    using ((__subject__ <= max));
+    }  errmessage := 
+        '{__subject__} must be no longer than {max} characters.';
+    annotation std::description := 
+        'Specifies the maximum length of subject string representation.';
+    ;
+
+.. image:: images/stdlib_search.png
+    :alt: A search result from the Schema Viewer page in the EdgeDB UI,
+          in which a search for the keywords max and val has turned up
+          three matching functions: max value, max ex value, and max
+          len value.
+    :width: 100%
```

Comment:
I like the addition of the image, but it feels a bit redundant to have the text output and then an image of the same text output. Maybe we ditch the text but flesh out the image alt a bit?

Reply:
Sure, have done that. Without making the alt text too long, I guess the most important point to add is that the search doesn't just show the function names but also their declarations which lets you see the internals and know exactly what the function behaviour will be.